.circleci/config.yml

```diff
@@ -0,0 +1,115 @@
+####################
+# CircleCI configuration reference:
+#   https://circleci.com/docs/2.0/configuration-reference
+####################
+# CircleCI built-in environment variables:
+#   https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables
+####################
+
+
+####################
+# Templates: see "anchors" in https://learnxinyminutes.com/docs/yaml/
+####################
+
+save_cache_settings: &save_cache_settings
+  key: python_mozetl-{{ .Branch }}-{{ .Environment.CIRCLE_WORKFLOW_ID }}
+  paths:
+    - ~/python_mozetl/.tox
```

Comment:
Would we benefit from saving [pip's cache dir](https://stackoverflow.com/a/41111916/1260237), too? Or is that not used in the tox case?